_includes/docs/user-guide/rule-engine-2-0/overview.md

```diff
@@ -82,97 +82,6 @@ Similar, if "Save to DB" call will succeed, the message will be successfully pro
 However, if the external system will not reply within 20 seconds, the message processing attempt will be marked as "timed-out".
 Similar, if "Save to DB" call will fail, the message will be marked as failed.
 
-#### Rule Engine Queue
```

Comment:
Please think about website indexing. We need to add the "redirect from" annotation to the new page (if it supports redirect from links with '#' ("/some_url#rule-engine-queue") or keep this sections and add a link instead of the text to the correct page.